source/support/send-email/index.html.md.erb

```diff
@@ -1,18 +1,17 @@
 ---
 title: Send an email to all GOV.UK Account users
-weight: 32
+weight: 33
```

Comment:
What is the thinking behind the weighting change?

Reply:
The "[Resolve Issues with GOV.UK Accounts](https://github.com/alphagov/govuk-account-team-manual/blob/main/source/support/resolve/index.html.md.erb)" content also has a weighting of 32. When multiple content files have the same weighting, every time the website loads, the pages could load in a different order. Additionally, having the weighting as different makes it easier to tweak the information hierarchy and add / remove content later.